src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleOptions2.cs

```diff
@@ -215,6 +215,12 @@ private static PerLanguageOption2<CodeStyleOption2<bool>> CreateQualifyAccessOpt
             "dotnet_style_prefer_is_null_check_over_reference_equality_method",
             $"TextEditor.%LANGUAGE%.Specific.{nameof(PreferIsNullCheckOverReferenceEqualityMethod)}");
 
+        internal static readonly PerLanguageOption2<CodeStyleOption2<bool>> PreferParameterNullChecking = CreateOption(
```

Comment:
The poor `AutomationObject` gets forgotten every time a new option is getting added :(
I believe this one should be added just like those:

https://github.com/dotnet/roslyn/blob/ab1062214006506b13f19bfb6215d84f7e71cb0f/src/VisualStudio/CSharp/Impl/Options/AutomationObject/AutomationObject.Style.cs#L234-L244

The effect of this is the user preference being persisted when importing/exporting VS settings.

Reply:
thanks!